crates/ruff_workspace/src/resolver.rs

```diff
@@ -156,9 +156,16 @@ impl<'a> Resolver<'a> {
             .insert(format!("{path}/{{*filepath}}"), self.settings.len() - 1)
         {
             Ok(()) => {}
-            Err(InsertError::Conflict { .. }) => {}
+            Err(InsertError::Conflict { .. }) => {
+                return;
+            }
             Err(_) => unreachable!("file paths are escaped before being inserted in the router"),
         }
+
+        // Insert a mapping that matches the directory itself (without a trailing slash).
+        // Inserting should always succeed because conflicts are resolved above and the above insertion guarantees
+        // that the path is correctly escaped.
+        self.router.insert(path, self.settings.len() - 1).unwrap();
```

Comment:
@ibraheemdev is there some wild card expression that we could use that I'm not aware of instead of inserting two routes (e.g. `{path}(/{{#filepath}})?`)

Reply:
No, this is the recommended way of doing it.